components/builder-scheduler/habitat/plan.sh

```diff
@@ -0,0 +1,75 @@
+pkg_name=hab-builder-scheduler
+pkg_origin=core
+pkg_version=undefined
+pkg_maintainer="Jamie Winsor <reset@chef.io>"
```

Comment:
We should just set this to `pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"`. I know it's not that way in the other plans but I think it makes more sense to just have our maintainer lines always read like that